### DISCUSSION TITLE
Breaking changes every week!

### DISCUSSION THREAD

**Discussion by jbenguira** [immich-app/immich#7405] [Q&A]
### The bug

Hello dear Immich team, 
would you consider publishing the breaking changes in Major versions instead of nearly every week recently?

### The OS that Immich Server is running on

Ubuntu 22.04

### Version of Immich Server

v1.95.0

### Version of Immich Mobile App

v1

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
-   image: tensorchord/pgvecto-rs:pg14-v0.1.11@sha256:0335a1a22f8c5dd1b697f14f079934f5152eaaa216c09b61e293be285491f8ee 
+   image: tensorchord/pgvecto-rs:pg14-v0.2.0@sha256:90724186f0a3517cf6914295b5ab410db9ce23190a2d9d0b9dd6463e3fa298f0
```

### Your .env content

```Shell
default
```

### Reproduction steps

```bash
docker pull every week == breaking change every week
```

**Comment by jrasm91** [comment]
No, we will bump major versions for breaking changes after going stable.

**Comment by bo0tzz** [comment]
You should not think of Immich's current versioning scheme as adhering to semver.
